### PR TITLE
Crucible | Rename cluster output to avoid conflicts with cluster variable

### PR DESCRIPTION
**Pull request by ramperher**
Test-Hints: assisted-abi

- While testing https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/92 in a disconnected environment, we could see there were conflicts with cluster variable and the output registered from some tasks, which was also called cluster. Renaming the output to avoid these conflicts.
- Fixing Ansible lint issues with some calls to roles that are made in the modified files.

### CODE REVIEW

**Comment by ramperher** [comment]
CC @tkrishtop

**Comment by ramperher** [comment]
I don't get the point of the failure in the `Pull Request / Ansible-lint Check (pull_request)`, since it's complaining about code that is not being changed in this PR. I suppose that, after a green job, it should be fine to merge this.

**Review by nocturnalastro — DISMISSED** [review]
LGTM

**Review by tkrishtop — CHANGES_REQUESTED** [review]
let's `git cherry-pick` your commit from the [main PR](https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/92/commits/dd1a271758858c11250b0c12925a3454408d533a) to avoid merge conflicts later.

**Comment by ramperher** [comment]
> let's `git cherry-pick` your commit from the [main PR](https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/92/commits/dd1a271758858c11250b0c12925a3454408d533a) to avoid merge conflicts later.

done

File: roles/add_day2_node/tasks/main.yml [hunk 1]
```diff
@@ -40,17 +54,15 @@
       ansible.builtin.import_role:
         name: redhatci.ocp.patch_host_config
       vars:
-        patch_cluster_id: "{{ add_host_cluster_id }}"
-        discovered_host: "{{ discovered_host_reply.json }}"
-        inventory_nodes:
```
Comment (ramperher) [inline roles/add_day2_node/tasks/main.yml:45]: inventory_nodes is not used in patch_host_config, that's why I've removed it.

File: roles/add_day2_node/tasks/main.yml [hunk 2]
```diff
@@ -40,17 +54,15 @@
...
-          - "{{ inventory_hostname }}"
+        patch_host_config_cluster_id: "{{ add_host_cluster_id }}"
+        patch_host_config_discovered_host: "{{ discovered_host_reply.json }}"
 
-    - name: "Wait for up to 20 minutes for {{ inventory_hostname }} to be ready"
+    - name: "Wait for up to 20 minutes to be ready - {{ inventory_hostname }}"
```
Comment (nocturnalastro) [inline roles/add_day2_node/tasks/main.yml:60]:
```suggestion
    - name: "Wait for up to 20 minutes host to be ready - {{ inventory_hostname }}"
```

Comment (ramperher) [reply roles/add_day2_node/tasks/main.yml:60]: done

**Comment by ramperher** [comment]
> * dci-check-change  : FAILURE tested on libvirt:assisted-abi
> * assisted-abi -> failure: https://www.distributed-ci.io/jobs/ff479cbf-101e-4686-bdd0-6449d560c608/jobStates

The job has failed in an issue that appears from time to time, I'll relaunch the job.

**Comment by ramperher** [comment]
I've reviewed everything again and my impression is that the issue in the debug job may be transient, I don't think it's related to this change. Also, assisted-abi is not using the roles I've changed in this PR, it's more for assisted on-prem. I'll test both options again, to see if it's finally passing.

**Review by nocturnalastro — APPROVED** [review]

**Comment by ramperher** [comment]
assisted-abi passed, waiting for a green assisted on-prem job (it's about to finish and it's passing now), then I'll rebase the change and merge

**Comment by ramperher** [comment]
I will merge since both assisted install methods have already been validated. Last change was just a rebase.

**Pull request merged**